### PR TITLE
File Added : Assignment9

### PR DESCRIPTION
**Pull request by urmilagagare**

### CODE REVIEW

File: urmila/HTML/Assignment9/Assignment9.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                <li><a href="home.html"><img src="d.png" alt="2"></li>
+                <li><a href="home.html"><img src="e.png" alt="2"></li>
+                <li><a href="home.html"><img src="f.png" alt="2"></li>
+        </ul>
+    </div>
+    <div class="state"><img src="g.png" alt="g"></div>
+</body>
+</html>
+</body>
+</html>
```
Comment (shubhamdalal49) [inline urmila/HTML/Assignment9/Assignment9.html]: remove unwanted extra tag from the end

File: urmila/HTML/Assignment9/Assignment9.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            <li><a href="home.html"><img src="2.png" alt="2"></a></li>
+            <li><a href="home.html"><img src="4.png" alt="4"></a></li>
+            <li><a href="home.html"><img src="5.png" alt="5"></a></li>
+        </ul>
+    </div>
+    <div class="background"><img src="3.png" alt="3"></div>
+    <div class="inner">
+        <form action="action.php">
+            <input type="text" id="fname" name="fname" placeholder="Enter skill, Designation etc..">
+            <span class="inpu">
```
Comment (shubhamdalal49) [inline urmila/HTML/Assignment9/Assignment9.html]: indentation error

File: urmila/HTML/Assignment9/Assignment9.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    <div class="jobs-list2">
+        <ul>
+                <li><a href="home.html"><img src="d.png" alt="2"></li>
+                <li><a href="home.html"><img src="e.png" alt="2"></li>
+                <li><a href="home.html"><img src="f.png" alt="2"></li>
+        </ul>
+    </div>
+    <div class="state"><img src="g.png" alt="g"></div>
+</body>
+</html>
```
Comment (shubhamdalal49) [inline urmila/HTML/Assignment9/Assignment9.html]: indentation error

File: urmila/HTML/Assignment9/style.css [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+width: 906px;
+height: 91px;
+left: 0px;
+top: 489px;
+}
+.jobs-list2{
+position: absolute;
+width: 906px;
+height: 91px;
+left: 0px;
```
Comment (shubhamdalal49) [inline urmila/HTML/Assignment9/style.css]: add space before the curly braces

File: urmila/HTML/Assignment9/style.css [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+top: 275px;
+}
+.display{
+position: absolute;
+width: 435px;
+height: 227px;
+left: 922px;
+top: 275px;
+}
+h1{
```
Comment (shubhamdalal49) [inline urmila/HTML/Assignment9/style.css]: add space before curly braces

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

**Review by Ashwin2812 — APPROVED** [review]

File: urmila/HTML/Assignment9/Assignment9.html [hunk 6]
```diff
@@ -38,20 +38,18 @@
...
-                <li><a href="home.html"><img src="e.png" alt="2"></li>
-                <li><a href="home.html"><img src="f.png" alt="2"></li>
+            <li><a href="home.html"><img src="d.png" alt="2"></li>
+            <li><a href="home.html"><img src="e.png" alt="2"></li>
+            <li><a href="home.html"><img src="f.png" alt="2"></li>
         </ul>
     </div>
     <div class="state"><img src="g.png" alt="g"></div>
 </body>
 </html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment9/Assignment9.html:55]: create proper indentation

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**